src/style-guide/README.md

```diff
@@ -321,14 +277,13 @@ By moving the `v-if` to a container element, we're no longer checking `shouldSho
 </ul>
 ```
 
-``` html
-<ul v-if="shouldShowUsers">
-  <li
-    v-for="user in users"
-    :key="user.id"
-  >
-    {{ user.name }}
-  </li>
+```html
+<ul>
+  <template v-for="user in users" :key="user.id">
```

Comment:
As I noted previously, this is technically a breach of the style guide's own recommendations for multi-attribute elements.

Personally I think it's fine.